spec/helper/revenuecat_internal_helper_spec.rb

```diff
@@ -214,6 +214,38 @@
 
       expect(File.read(file_to_update_on_latest_stable_release_5)).to eq('Contains version: 1.2.4')
     end
+
+    it 'does update files on latest stable release if previous version does not match previous version given' do
+      File.write(file_to_update_on_latest_stable_release_5, '<meta http-equiv="refresh" content="0; url=https://sdk.revenuecat.com/android/7.10.7/index.html" />')
+
+      Fastlane::Helper::RevenuecatInternalHelper.replace_version_number(
+        '1.2.3',
+        '1.2.4',
+        {},
+        {},
+        {
+          file_to_update_on_latest_stable_release_5 => ["url=https://sdk.revenuecat.com/android/{x}/index.html"]
+        }
+      )
+
+      expect(File.read(file_to_update_on_latest_stable_release_5)).to eq('<meta http-equiv="refresh" content="0; url=https://sdk.revenuecat.com/android/1.2.4/index.html" />')
```

Comment:
not super against this, but it feels like we could make this a bit easier to parse by either string formatting the version or breaking the expected and actual strings into their own variables and then comparing them? 
like
```suggestion
      original = '<meta http-equiv="refresh" content="0; url=https://sdk.revenuecat.com/android/7.10.7/index.html" />'
      expected = '<meta http-equiv="refresh" content="0; url=https://sdk.revenuecat.com/android/1.2.4/index.html" />'
      actual = File.read(file_to_update_on_latest_stable_release_5)
      expect(actual).to eq(expected)
```

(or use string formatting to make the version a variable)